packages/gcf-utils/package.json

```diff
@@ -27,6 +27,7 @@
     "@types/express": "^4.17.0",
     "@types/ioredis": "^4.14.8",
     "@types/lru-cache": "^5.1.0",
+    "@types/sonic-boom": "^0.7.0",
```

Comment:
gcf-utils had problems on the main branch do to exporting `@types/sonic-boom`, but only having it as dev dep.